### ISSUE TITLE
Get node power status

### ISSUE THREAD

**Issue by naved001** [CCI-MOC/obmd#12]
I think it would be a useful to include support for getting node power status. Once we have it here, we can expose it in HIL too. 

```
[root@kumo-hil ~]# ipmitool -I lanplus -H 10.1.10.1 -U root -P password chassis power status
Chassis Power is off
[root@kumo-hil ~]# ipmitool -I lanplus -H 10.1.10.1 -U root -P password chassis power on
Chassis Power Control: Up/On
[root@kumo-hil ~]# ipmitool -I lanplus -H 10.1.10.1 -U root -P password chassis power status
Chassis Power is on
```

https://github.com/CCI-MOC/hil/issues/724

**Comment by zenhack** [comment]
Sounds good to me. Should be pretty easy.

**Comment by Izhmash** [comment]
Will I need to do some sort of stdout redirection to get the ipmitool output from `ipmitool -I lanplus -H 10.10.0.41 -U admin -P Xxu46RCjtxiLdc chassis power status`?  I've written a driver function that calls it but getting the output is tricky.

**Comment by zenhack** [comment]
Look at what we're doing for the console streaming; the same logic should do the trick.

**Comment by zenhack** [comment]
Also, It may not need the pty hack; I don't know whether ipmitool insists on being attached to a terminal for operations other than sol. Have a look also at:

https://golang.org/pkg/os/exec/

If you have more specific issues you're bumping into getting the output, describe them and I can try to help.

**Comment by Izhmash** [comment]
I've got a working prototype of this here: https://github.com/CCI-MOC/obmd/compare/master...Izhmash:node-power-status

I've been testing the power status on real nodes.  I may have included too much of the console dialing functionality, so perhaps there is a better way to do this.

**Comment by zenhack** [comment]
You shouldn't need to stream the results; just read the whole output up front, wait for the process to stop, and return the result. Probably with os/exec.Cmd.Output, though if it insists on being attached to a pty, you can use something like io/ioutil.ReadAll. GetPowerStatus() should just return a string, which will simplify the entire call chain.

You also don't need to add an extra channel to the server for these calls; just use RunInServer like e.g. PowerCycle does.

**Comment by Izhmash** [comment]
Addressed by #16
